### PR TITLE
changed Return into Return or Enter

### PR DESCRIPTION
**Pull request by ilanchezhian-m**
### Description

Changing the Return into Return or Enter for better understanding especially for beginners

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
My suggestion:

1. We should prefer to use <kbd>Enter</kbd> everywhere.
2. We should only mention <kbd>Return</kbd> as a possible alternative to <kbd>Enter</kbd> the first time: "<kbd>Enter</kbd> (or <kbd>Return</kbd>, depending on your keyboard)", and then avoid writing both alternatives everywhere.

**Comment by ilanchezhian-m** [comment]
so your suggestion is to change the first line to "(Enter or Return)", and the remaining lines stay the same which is a return, or else change return into enter in the remaining lines too?

File: files/en-us/web/javascript/guide/introduction/index.md [hunk 1]
```diff
@@ -101,10 +101,10 @@ console.log(eval("3 + 5"));
 
 ### Multi-line input in the console
 
-By default, if you press <kbd>Return</kbd> after entering a line of code, then the string you typed is executed. To enter multi-line input:
+By default, if you press <kbd>Return or Enter</kbd>( after entering a line of code, then the string you typed is executed. To enter multi-line input:
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/introduction/index.md:104]:
```suggestion
By default, if you press <kbd>Enter</kbd> (or <kbd>Return</kbd>, depending on your keyboard) after entering a line of code, then the string you typed is executed. To enter multi-line input:
```

Comment (ilanchezhian-m) [reply files/en-us/web/javascript/guide/introduction/index.md]: Thanks, I applied the suggestion !

**Review by Josh-Cena — APPROVED** [review]
LGTM. Thanks 😄

**Comment by Josh-Cena** [comment]
Congratulations on your first contribution here—welcome aboard!

**Comment by ilanchezhian-m** [comment]
Thank you so much!

**Pull request merged**